drivers/serial/Kconfig

```diff
@@ -17,6 +17,12 @@ config NRF_SW_LPUART_MAX_PACKET_SIZE
 	int "Maximum RX packet size"
```

Comment:
When the interrupt driven API is used, this option specifies also the TX buffer size. Perhaps this option's description should be updated.

Reply:
i've rearranged kconfig a bit, please take a look.